cypher/cypher-docs/src/docs/dev/ql/administration/security/database/admin-database-syntax.asciidoc

```diff
@@ -70,13 +70,13 @@ GRANT ALL [[DATABASE] PRIVILEGES]
 | Grant the specified roles all privileges for the default database, specific database(s), or all databases.
 
 | [source, cypher]
-GRANT {SHOW \| TERMINATE} TRANSACTION[S] [(* \| user[, ...])]
+GRANT {SHOW \| TERMINATE} TRANSACTION[S] ["(" {* \| user[, ...]} ")"]
```

Comment:
`(` is not a special character so needs no escaping, not really sure though how to make it look not cluttered...
```suggestion
GRANT {SHOW \| TERMINATE} TRANSACTION[S] [( {* \| user[, ...]} )]
```